src/isotools/_transcriptome_stats.py

```diff
@@ -758,5 +761,141 @@ def pairwise_event_test(e1, e2, transcriptome, gene, min_dist=1, test="chi2"):
         p_value = test_res[1]
         test_stat = test_res[0]
 
-        return p_value, test_stat, priA_priB, priA_altB, altA_priB, altA_altB
+        return p_value, test_stat, int(priA_priB-0.01), int(priA_altB-0.01), int(altA_priB-0.01), int(altA_altB-0.01)
+
+
+def gene_coordination_test(self, gene, test="chi2", min_dist=1, min_total=100,
```

Comment:
This should be a function of the gene

Reply:
I made it a function of gene coordination test and change everything that needed to be changed to make it work.